# 0xF7 : Status Message - Operation Information

# Variables declaration
	uint8 SyncByte  # IMU Address
	uint8 MessageID  # Message ID
	uint32 hours_of_operation  # Hours of Operation
	uint32 boot_count  # Boot Count
	uint16 Checksum  # Uint16 Checksum. Complement = False
